packages/kit/src/exports/vite/index.js

```diff
@@ -128,7 +128,7 @@ const warning_preprocessor = {
  * rather than relative to this package
  * @param {string} dependency
  */
-async function resolve_peer_dependency(dependency) {
+function resolve_peer_dependency(dependency) {
```

Comment:
Should we move this out into a util and reuse it in the other file or not worth it?

Reply:
yeah, i was just being lazy